Sources/TuistKit/Cache/CacheController.swift

```diff
@@ -148,10 +149,23 @@ final class CacheController: CacheControlling {
             }
 
             // Build
-            try buildAndCacheFramework(path: projectPath, target: target, configuration: cacheProfile.configuration, hash: hash)
+            do {
+                try buildAndCacheFramework(path: projectPath, target: target, configuration: cacheProfile.configuration, hash: hash)
+            } catch let error {
+                errors[target.target.name] = error
+            }
+        }
+        
+        if errors.isEmpty {
+            logger.notice("All cacheable targets have been cached successfully as \(artifactBuilder.cacheOutputType.description)s", metadata: .success)
+        } else {
+            logger.notice(
+                """
+                Not all cacheable targets have been cached successfully as \(artifactBuilder.cacheOutputType.description)s! Errors:\n\(errors.map({ (target: String, error: Error) in
+                    "\(target): \(error.localizedDescription)\n"
+                    }))
+                """, metadata: .error)
```

Comment:
Could you provide more context on this change? What this has to do with the excluding resources stuff?